cloud-computing/gramine/README.md

```diff
@@ -0,0 +1,76 @@
+## How to run demo with Gramine on localhost
+
+1. Start a SPS container:
+```shell
+SPS_VERSION=<set SPS version here>
+docker run -d -v /opt/multiple/sessions:/graphene/workplace/sessions -v /opt/secret-prov/certs/:/graphene/workplace/certs -p 8080:8080 -p 4433:4433 -e SPS_USERNAME=admin -e SPS_PASSWORD=admin --name iexec-sps iexechub/iexec-sps:${SPS_VERSION}
```

Comment:
Could we add some '\\' to improve readability on HTML rendered README ?

Reply:
Right, see https://github.com/iExecBlockchainComputing/python-hello-world/pull/13/commits/a564cac5a63b703ad27d7d25181f85fc03494137.